Wikibase/Sniffs/Commenting/DisallowedDocTagsSniff.php

```diff
@@ -32,6 +32,9 @@ class Wikibase_Sniffs_Commenting_DisallowedDocTagsSniff implements PHP_CodeSniff
 		// https://phpunit.de/manual/current/en/appendixes.annotations.html#appendixes.annotations.covers
 		'@cover' => '@covers',
 		'@use' => '@uses',
+
+		// https://github.com/slevomat/coding-standard#slevomatcodingstandardcommentingforbiddenannotations-
```

Comment:
To be honest, I disagree with almost all the other tags listed there :)

- > `@author`, `@created`, `@version`: we have version control systems.

  Sure, but every `git blame` I’ve ever done on Wikibase-related code has involved at least three iterations of “copy hash” → `git show HASH` → “oh, I don’t care about this one” (whitespace, `array()` → `[]`, other minor stuff) → `git blame FILE HASH^` → repeat. `@since` is much faster.

- > `@copyright`, `@license`: it's not necessary to repeat licensing information in each file.

  Well, at least there are many people more knowledgeable about copyright law than me who disagree :)

- > `@throws`: it's not possible to enforce this annotation and the information can become outdated.

  With exactly the same argument, `@param` and `@return` could be forbidden as well.

That said, the way we used to use `@package` in WBQC really was completely redundant, so it makes sense to disallow it.

Reply:
All this is unrelated to this patch, but since we are here:
* The author is not arguing against `@since`. ;-) I like `@since`, but I agree `@version` and `@created` are cruft.
* I don't think I need to be "knowledgeable about copyright law" to see that copy-pasting a "magical" text snippet a billion times is not going to do anything but making people actively ignore it.
* I agree with `@throws` in so far that it is quite pointless when there is no comment that explains what it means, and how to deal with it.